docs/source/install.md

```diff
@@ -0,0 +1,137 @@
+# Install Fabric Samples
+
+Please install the [Prerequisites](./prereqs.html) before following these install instructions.
+
+**Fabric-Samples:**<br>
```

Comment:
If we consolidate the platform sections, we can make these the sections of this article:
- Fabric docker images and Fabric samples
- Fabric source code

Reply:
@nikhil550 I can, but the slashes would be wrong.  I mean it would be obvious to windows users, but still forward vs backward slashes would be incorrect.

 Also, doesn't Windows use different variables for $HOME ?

>> $ mkdir -p $HOME/go/src/github.com/<your_github_userid>
>> $ cd $HOME/go/src/github.com/<your_github_userid>

However, if we remove the instructions to install the files in this directory altogether, then collapsing makes complete sense.  And technically you are right, we don't need to install Samples here, but I had the Fabric code base as an optional install in this page which made following the same directory structure for both repos $HOME/go/src/github.com/<your_github_userid> more efficient.  

But I have since removed the Fabric section, so . . . we could just delete it and collapse it.

Reply:
I've collapsed the platform installs into a single download section, and added a few platform-specific considerations at the end.